assets/messages.yml

```diff
@@ -1039,6 +1039,19 @@ messages:
       shortcut: qul
       message: "%quick_links%"
       fillname: []
+  removed-version:
+    - project:
+        - mc
+        - mce
+        - mcl
+        - mcpe
+      name: Panel - Removed Extra Version
```

Comment:
Maybe something like "Panel - Removed resolved issue Affected Version" or similar would be clearer.